governance/program/src/state.rs

```diff
@@ -0,0 +1,281 @@
+//! Program accounts
+
```

Comment:
nit: all the types in this file should be `#[repr(C)]` so the state layout is well known for non-Rust clients

Reply:
Done